app/scripts/components/analysis/results/chart-card.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import {
+  Toolbar,
+  ToolbarIconButton,
+  VerticalDivider
+} from '@devseed-ui/toolbar';
+import {
+  CollecticonCircleInformation,
+  CollecticonDownload2
+} from '@devseed-ui/collecticons';
+
+import {
+  CardSelf,
+  CardHeader,
+  CardHeadline,
+  CardTitle,
+  CardActions,
+  CardBody
+} from '$components/common/card';
+import { TimeseriesData } from './timeseries-data';
+import { ChartLoading } from '$components/common/loading-skeleton';
+import { ChartCardAlert, ChartCardNoData } from './chart-card-message';
+import Chart from '$components/common/chart/analysis';
+import { dateFormatter } from '$components/common/chart/utils';
+
+interface ChartCardProps {
+  title: React.ReactNode;
+  chartData: TimeseriesData;
+}
+
+export default function ChartCard(props: ChartCardProps) {
+  const { title, chartData } = props;
+  const { status, meta, data, error } = chartData;
+
+  return (
+    <CardSelf>
+      <CardHeader>
+        <CardHeadline>
+          <CardTitle>{title}</CardTitle>
+        </CardHeadline>
+        <CardActions>
+          <Toolbar size='small'>
+            <ToolbarIconButton variation='base-text'>
+              <CollecticonDownload2 title='Download' meaningful />
+            </ToolbarIconButton>
+            <VerticalDivider variation='dark' />
+            <ToolbarIconButton variation='base-text'>
+              <CollecticonCircleInformation title='More info' meaningful />
+            </ToolbarIconButton>
+          </Toolbar>
+        </CardActions>
+      </CardHeader>
+      <CardBody>
+        {status === 'errored' && <ChartCardAlert message={error.message} />}
+
+        {status === 'loading' ? (
+          meta.total ? (
+            <ChartLoading message={`${meta.loaded} of ${meta.total} loaded.`} />
+          ) : (
+            <ChartLoading message='Loading...' />
+          )
+        ) : null}
+
+        {status === 'succeeded' ? (
+          data.timeseries.length ? (
+            <Chart
+              timeSeriesData={data.timeseries}
+              uniqueKeys={[
+                { label: 'Min', value: 'min', active: true },
+                { label: 'Max', value: 'max', active: true },
+                { label: 'STD', value: 'std', active: true }
+              ]}
+              xKey='date'
+              dates={data.timeseries.map((e) =>
+                dateFormatter(new Date(e.date), '%Y/%m')
+              )}
+              dateFormat='%Y/%m'
+              altTitle='alt title'
+              altDesc='alt desc'
+              xAxisLabel='x axis label'
+              yAxisLabel='y axis label'
```

Comment:
Are we able to address that now? (I think I recall we don't have any anything about units)

Reply:
Added a more generic description for now